Makefile

```diff
@@ -34,9 +34,9 @@ ifdef EMSCRIPTEN
 	OUTPUT=x16emu.html
 endif
 
-OBJS = cpu/fake6502.o memory.o disasm.o video.o ps2.o via.o loadsave.o spi.o vera_spi.o sdcard.o main.o debugger.o javascript_interface.o joystick.o
+OBJS = cpu/fake6502.o memory.o disasm.o video.o ps2.o via.o loadsave.o spi.o vera_spi.o sdcard.o main.o debugger.o javascript_interface.o joystick.o icon.o
 
-HEADERS = disasm.h cpu/fake6502.h glue.h memory.h video.h ps2.h via.h loadsave.h joystick.h
+HEADERS = disasm.h cpu/fake6502.h glue.h memory.h video.h ps2.h via.h loadsave.h joystick.h icon.h
 
```

Comment:
Add a rule to generate `icon.c` and `icon.h` as following:
```
icon.c icon.h: icon.bmp tools/embed_c.py
	python3 tools/embed_c.py icon.bmp -o icon.c -h icon.h -n icondata
```
That way, there is no need to add icon.c and icon.h to the repo.

Add `icon.c` and `icon.h` to `.gitignore`

Reply:
I've updated the Makefile to generate the header files during the build.  This does introduce a build dependency on python however.